packages/backend/src/helpers/graphql-instance.ts

```diff
@@ -21,11 +21,21 @@ function ApolloServerPluginUserTracer(): ApolloServerPlugin<AuthenticatedContext
     async requestDidStart(requestContext) {
       // Add the tag right before we reply the user.
       // https://www.apollographql.com/docs/apollo-server/integrations/plugins#request-lifecycle-event-flow
+      const isAdminQuery = requestContext.contextValue?.isAdminQuery
+        ? 'true'
+        : 'false'
+      tracer.scope().active()?.addTags({
+        isAdminQuery,
+      })
+
       const currentUser = requestContext.contextValue?.currentUser
       if (currentUser) {
         tracer.setUser({
           id: currentUser.id,
           email: currentUser.email,
+
+          // For convenience
+          isAdminQuery,
```

Comment:
do we want to set if currentUser is null as well?

Reply:
already set in span